code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -711,6 +711,16 @@
 	icon_state = "female_blackalt"
 	gender = FEMALE
 
+/datum/sprite_accessory/underwear/female_white_neko
+	name = "Ladies White Neko"
+	icon_state = "female_neko_white"
+	gender = FEMALE
+
+/datum/sprite_accessory/underwear/female_black_neko
+	name = "Ladies Black Neko"
+	icon_state = "female_neko_black"
+	gender = FEMALE

```

Comment:
underpants, even ones intended specifically for women, need to be able to fit on men.

otherwise their dixel hangs out.

Try and model it on a man and just add the pixels that poke through, they'll automatically be removed for women